### PR TITLE
Added WikiText-TL-39

### PR DESCRIPTION
**Pull request by jcblaisecruz02**
This PR adds the WikiText-TL-39 Filipino Language Modeling dataset.

Paper: https://arxiv.org/abs/1907.00409
Repo: https://github.com/jcblaisecruz02/Filipino-Text-Benchmarks

### CODE REVIEW

**Comment by lhoestq** [comment]
looks like this PR also includes changes about another dataset `covid_qa_deepset`

Could you create another branch and another PR that only includes the changes for the wikitext-tl-39 dataset ?

File: datasets/wikitext_tl_39/wikitext_tl_39.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        super(WikitextTL39Config, self).__init__(
+            version=datasets.Version(
+                "1.0.0",
+            ),
+            **kwargs,
+        )
+        self.data_url = data_url
+
+
+class WikitextTL39(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/wikitext_tl_39/wikitext_tl_39.py:48]:
```suggestion
class WikitextTl39(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name.

Can you also move the dataset directory from `./datasets/wikitext_tl_39` to `./datasets/wikitext_tl39` instead, and rename the script accordingly ? This way the camel case and snake case names match.

**Review by lhoestq — COMMENTED** [review]
Really cool !

I just left a comment about the names of the dataset class and the dataset script name.
Can you do this change please ?

After this change you'll need to regenerate the daatset_infos.json file

File: datasets/wikitext_tl_39/wikitext_tl_39.py (hunk 1 above)
Comment (jcblaisecruz02) [reply datasets/wikitext_tl_39/wikitext_tl_39.py:48]: Gotcha! I'll work on the changes then start a new branch. Thanks!

**Pull request closed without merging**